lib/node/actions/http_client.dart

```diff
@@ -6,12 +6,15 @@ import 'dart:js_interop';
 
 @JS()
 inline class HttpClient {
+  // TODO: This needs to be assigned somewhere.
```

Comment:
Adding this representation class field addresses the compilation issue, and the tests for this repo pass (all the `setup-dart` scenarios work), but it's a bit disconcerting that nothing assigns to this field.

Reply:
The ergonomics of inline classes can be a bit weird. This field *is* initialized, but by the `external` constructor implicitly. If extension types + the primary constructor syntax is finalized, this becomes a little less confusing.

Reply:
Gotcha - sounds like not having explicit initialization isn't an issue.